data/filetype_extensions.conf

```diff
@@ -7,7 +7,7 @@ Abc=*.abc;*.abp;
 ActionScript=*.as;
 Ada=*.adb;*.ads;
 Asciidoc=*.asciidoc;*.adoc;
-ASM=*.asm;
+ASM=*.asm;*.asm51;*.a51;*.asm;*.ll;*.s03;*.s79;*.s82;*.s90;*.s;
```

Comment:
Whoops. I bet that duplicate mistake does not give me a good first-impression on this team. (^u^)

*.ll is LLVM assembly.

From now on, I will be discuss my intended commits and ideas on the Geany-dev mailing list before submitting contributions. Hopefully, that will save Geany's GitHub team some time and inconvenience. I will also include more detailed commit notes and be more detailed in explanations.